onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -483,6 +488,8 @@ class PlannerImpl {
       UseCount(initializer_name)++;
     }
 
+    std::unordered_set<OrtValueIndex> node_arg_has_explicit_consumer;
```

Comment:
Rename this to node_arg_has_explicit_consumer_set or set_node_arg_has_explicit_consumer.

Reply:
Renamed